README.md

```diff
@@ -48,10 +48,8 @@ dependencies:
 import 'package:countly_flutter/countly_flutter.dart';
 
 void main() {
-  // If you want to catch Dart errors, run your app inside a Zone and pass Countly.recordDartError as the onError parameter to it.
-  runZonedGuarded<void>(() {
-    runApp(MaterialApp(home: const MyApp()));
-  }, Countly.recordDartError);
+  // If you want to catch Dart errors, call enableCrashReporting before you initialize your Countly SDK.
```

Comment:
previously catching dart errors needed extra effort and it was noted. Is it still the same? If not why we mention Dart errors here.?